mmdet3d/models/backbones/multi_backbone.py

```diff
@@ -0,0 +1,125 @@
+import copy
+import torch
+from mmcv.cnn import ConvModule
+from mmcv.runner import load_checkpoint
+from torch import nn as nn
+
+from mmdet.models import BACKBONES, build_backbone
+
+
+@BACKBONES.register_module()
+class MultiBackbone(nn.Module):
+    """MultiBackbone with different config. '''.
+
+    Args:
+        num_streams (int): The number of backbones.
+        backbones (list or dict): A list of backbone configs.
```

Comment:
We may add an additional arguments `suffixs = ['net0', 'net1']` thus the config used for backbones is just the same as the single backbone for a more clear usage.